hw/ip_templates/pinmux/rtl/pinmux.sv.tpl

```diff
@@ -613,6 +618,13 @@ module pinmux
   // 1, respectively.
   assign mio_wkup_mux = AlignedMuxSize'({mio_wkup_no_scan, 1'b1, 1'b0});
 
+% if n_dio_pads > n_mio_pads + 2:
+  logic unused_mio_wkup_mux_signals;
+  assign unused_mio_wkup_mux_signals = ^{mio_wkup_mux[(AlignedMuxSize - 1):(NMioPads + 2)]};
+  logic unused_dio_wkup_mux_signals;
+  assign unused_dio_wkup_mux_signals = ^{dio_wkup_mux[(AlignedMuxSize - 1):(NMioPads + 2)]};
+
+% endif
```

Comment:
After taking another look at this I think this actually doesn't make sense. Either you have more MIOs or more DIOs, so it should only be required to waive either unused DIOs or unused MIOs but not both.